### PR TITLE
chore: update libp2p

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

**Review by willscott — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -175,8 +162,9 @@ require (
 	golang.org/x/mod v0.4.2 // indirect
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
 	golang.org/x/text v0.3.7 // indirect
-	golang.org/x/tools v0.1.2 // indirect
+	golang.org/x/tools v0.1.5 // indirect
 	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	lukechampine.com/blake3 v1.1.6 // indirect
```
Comment (AnomalRoil) [inline go.mod:169]: Not going to actually complain, but I would very much like to know why and where Blake3 is being used 😅

Comment (willscott) [reply go.mod:169]:
```
$ go mod why lukechampine.com/blake3
...
github.com/ipfs/go-cid
github.com/multiformats/go-multihash
github.com/multiformats/go-multihash/register/all
github.com/multiformats/go-multihash/register/blake3
```

**Comment by AnomalRoil** [comment]
> The base branch requires all commits to be signed 

Would you mind rebasing and signing your commit?

```
git rebase --exec 'git commit --amend --no-edit -n -S' -i HEAD~1
```
should do the trick, if you've got a git signing key configured already and in your Github profile.

**Comment by Stebalien** [comment]
Done.

**Pull request merged**